global.R

```diff
@@ -0,0 +1,43 @@
+###### MEtaInsight ######
+
+library(BUGSnet)
```

Comment:
I like that these are in alphabetical order, but it's recommended that plyr is loaded before dplyr https://www.r-project.org/nosvn/pandoc/dplyr.html#:~:text=Plyr%20compatibility&text=I%27d%20recommend%20loading%20plyr,be%20faster%20and%20more%20general.

Reply:
I've split that one library out of the alphabetical list for that reason. This is where explicit scoping would make that irrelevant, but that's obviously a big job to scope the whole codebase.